internal/engine/upper.go

```diff
@@ -935,6 +935,7 @@ func handleDockerComposeLogAction(state *store.EngineState, action DockerCompose
 
 	dcState, _ := ms.ResourceState.(dockercompose.State)
 	ms.ResourceState = dcState.WithCurrentLog(model.AppendLog(dcState.CurrentLog, action, state.LogTimestamps))
```

Comment:
this should also be on the CombinedLog right?

Reply:
@nicks [`state.Log`](https://github.com/windmilleng/tilt/blob/4f0e74b987d728f1cde3ce10cabb75a1a6f5666f/internal/store/engine_state.go#L55) is the combined log?

Reply:
here's an example of what i mean:
https://github.com/windmilleng/tilt/blob/master/internal/engine/upper.go#L777
why doesn't this code the same as that?

Reply:
Ah I see, I misunderstood what CombinedLog meant. Thanks!